runtimes/runtimes/util/testingUtils.ts

```diff
@@ -0,0 +1,38 @@
+import sinon from 'sinon'
+import {
+    Logging,
+    Lsp,
+    IdentityManagement,
+    Telemetry,
+    Workspace,
+    CredentialsProvider,
+    Chat,
+    Runtime,
+    Notification,
+} from '../../server-interface'
+
+export function createStubFromInterface<T>(): sinon.SinonStubbedInstance<T> & T {
+    const stub = {} as sinon.SinonStubbedInstance<T> & T
+    return new Proxy(stub, {
+        get: (target, property) => {
+            if (property in target) {
+                return target[property as keyof typeof target]
+            }
+            const method = sinon.stub()
+            ;(target as any)[property] = method
+            return method
+        },
+    })
+}
+
+export type Features = {
```

Comment:
Let's define it as part of server interface? We keep creating this type/interface in every new place.

Reply:
Moved `Features` to `/server-interface/server.ts`.